prime-router/metadata/schemas/covid-19.schema

```diff
@@ -267,7 +267,11 @@ elements:
 
   - name: ordering_facility_country
     type: TEXT
+    natFlatFileField: Ordering_facility_country
+    hhsGuidanceField:
+    hl7Field: ORC-22-6
     default: USA
```

Comment:
Just a note here that because of this default that already existed, all HL7 receivers will now be getting the ORC-22-6 field filled even if they weren't previously. I don't think it's a big deal as the ORC-22 is a pretty universally accepted field, but we could potentially see some receiver ask us to blank that field as a result of this.

Reply:
That is a very good point.  We need to talk about this in 16 mins, indeed.